src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -452,9 +465,18 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
                   "https",
                   userAgentSuffix,
                   gzipDisabled);
+          logger.info("Driver OCSP mode: {}, gzip disabled: {}, proxy protocol: {}," +
```

Comment:
there is already an identical log above - do we want to put it in a single method?

Reply:
Done.